modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -87,15 +88,29 @@ public BrowserWebDriverContainer() {
     /**
      * Constructor taking a specific webdriver container name and tag
      * @param dockerImageName Name of the docker image to pull
+     * @deprecated use {@link BrowserWebDriverContainer(DockerImageName)} instead
      */
+    @Deprecated
     public BrowserWebDriverContainer(String dockerImageName) {
-        this();
-        super.setDockerImageName(dockerImageName);
+        this(new DockerImageName(dockerImageName));
         this.customImageNameIsSet = true;
         // We have to force SKIP mode for the recording by default because we don't know if the image has VNC or not
         recordingMode = VncRecordingMode.SKIP;
     }
 
+    public BrowserWebDriverContainer(DockerImageName dockerImageName) {
```

Comment:
`this.customImageNameIsSet = true;` and `recordingMode = VncRecordingMode.SKIP;` from `BrowserWebDriverContainer(String)` are missing

Reply:
Fixed